robot/basestation/static/js/elements/feed/CmdInput.js

```diff
@@ -0,0 +1,58 @@
+$(document).ready(() => {
+  const send_cmd = $('#send-serial-cmd')
+  const cmd_input = $('#serial-cmd-input')
+  const invalid_cmd_msg = 'invalid command, please check input and try again'
+  const invalid_module_msg = 'Invalid module, please input module before command. Valid modules are rover, arm, ik and pds'
+
```

Comment:
Your string constants should have their variable name in CAPS

Reply:
Fixed